include/dca/phys/dca_step/cluster_solver/ctint/structs/solver_configuration.hpp

```diff
@@ -46,54 +49,29 @@ class SolverConfiguration : public MatrixConfiguration {
   template <class RngType>
   void insertRandom(RngType& rng);
 
-  // TODO: pass output as argument.
+  // Returns at most two candidates. -1 stands for a missing candidate.
```

Comment:
well now it always returns an array of size 2.  So its not going to be -1, update the documentation please.
Documenta removal_rand.

Reply:
Updated the comment. I want something that returns up to two indices, with no allocation, so I return -1 for a missing index.

Reply:
Ah I see for either one.